publiccode.yml

```diff
@@ -0,0 +1,145 @@
+      directly between a service consumer and a service provider. 
+
+      Message routing is based on organization and service level identifiers
+      that are mapped to physical network locations of the services by X-Road.
+      All the evidence regarding the data exchange is stored locally by the data
+      exchange parties, and no third parties have access to the data.
+      Time-stamping and digital signature together guarantee non-repudiation of
+      the data sent via X-Road. The logs provided by X-Road can be used in a
+      court proceeding as evidence.
+
+      ### Authorization Framework
+
+      X-Road implements an authorization framework that is used to manage access
+      rights to services. Access rights management is based on the organization
+      and service level identifiers. 
+
+      The key idea of X-Road is that each service provider owns its data and is
+      responsible for managing access rights of its services. In other words,
+      publishing service to X-Road does not mean that the service is
+      automatically accessible to all X-Road member organizations. Usually,
+      access rights are granted on the information system level – a service
+      provider grants a specific information system access to a service.
+
+      ### Monitoring and Reporting
+
+      X-Road provides monitoring and reporting capabilities that can be used to
+      collect operational reporting data and technical monitoring information
+      from the ecosystem. The information can be used to measure the usage of
+      individual services, understand dependencies and relationships between
+      different information systems and services, monitor service health,
+      monitor used X-Road software versions, etc. Each X-Road member
+      organization can access its own data, whereas the X-Road operator can
+      access all the members’ data.
+
+      ### Cross-border Data Exchange
+
+      X-Road provides built-in support for cross-border data exchange through
+      federation, which means joining together two X-Road ecosystems. Members of
+      the federated ecosystems can publish and consume services with each other
+      as if they were members of the same ecosystem.
+
+      It is possible to create federation connections with multiple ecosystems,
+      but transitive federation relationships are not supported. An ecosystem
+      does not have a federation relationship with another ecosystem that it's
+      not directly federated with.
+
+    shortDescription: |-
+      X-Road® is open-source software that provides unified and secure data
+      exchange between organisations in a collaborative ecosystem.
+    videos:
+      - 'https://youtu.be/EvhssW_Uv8s'
+developmentStatus: stable
+legal:
+  license: MIT
+  mainCopyrightOwner: Nordic Institute for Interoperability Solutions (NIIS)
+  repoOwner: Nordic Institute for Interoperability Solutions (NIIS)
+localisation:
+  availableLanguages:
+    - en
+    - ee
```

Comment:
The ISO 639-2 language code for Estonian should be `et`: https://www.loc.gov/standards/iso639-2/php/langcodes_name.php?code_ID=133

Reply:
Fixed. It was a mix up with the country code.